README.md

```diff
@@ -12,3 +13,19 @@ this repository:
 Or directly from our seed node:
 
     cargo install --force --locked --git https://seed.alt-clients.radicle.xyz/radicle-client-tools.git radicle-tools
+
+### From an APT repository on Debian/Ubuntu
+
+```
+curl https://europe-north1.pkg.dev/doc/repo-signing-key.gpg | sudo apt-key add -
+echo 'deb https://us-central1-apt.pkg.dev/projects/radicle-services radicle-tools main' | sudo tee -a  /etc/apt/sources.list.d/artifact-registry.list
+sudo apt update
+sudo apt install radicle-tools
+```
+
+### From Homebrew on macOS
+
+```
+brew tap radicle-dev/radicle-client-tools https://github.com/radicle-dev/radicle-client-tools.git
+brew install radicle-dev/radicle-client-tools/radicle-tools
```

Comment:
Can we (1) use our seed node directly, and (b) git rid of this `radicle-dev` prefix? maybe just `radicle`?

Reply:
yep, done

Reply:
Cool, you forgot the (1) though :D

Reply:
Ugh, I didn't push it somehow 😬